src/vs/workbench/contrib/welcomeGettingStarted/browser/featuredExtensionService.ts

```diff
@@ -59,10 +62,10 @@ export class FeaturedExtensionsService extends Disposable implements IFeaturedEx
 
 		await this._init();
 
-		let treatments = this.treatment?.extensions.filter(extension => !this.ignoredExtensions.has(extension)) ?? new Array<string>();
+		let treatments = this.treatment.extensions.filter(extension => !this.ignoredExtensions.has(extension)) ?? new Array<string>();
 		const featuredExtensions: IFeaturedExtension[] = new Array();
 
-		if (this.treatment?.showAsList !== 'true' && treatments.length > 0) {
+		if (this.treatment.showAsList !== 'true' && treatments.length > 0) {
```

Comment:
This scans a little funny, is this always guaranteed to be a string?

Reply:
yeah it's always a string. 
This and the extension ids get sent as a part of the same string. Since here is the only place I check for this, it felt easier to leave it as a string instead of converting it into a bool.